mitodl/ocw-to-hugo
# Fix code coverage

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #182 

#### What's this PR do?
Adjusts a few things to get code coverage working again

#### How should this be manually tested?
You should be able to run `npm run coverage` and see a new directory `coverage` which has some HTML files showing coverage of the files affected by the unit tests. You should also see that the code uploaded to codecov in the github check, though you may not be able to actually view it since I get a 400 error viewing that coverage. Hopefully that goes away in future PRs.
